### ISSUE TITLE
Staging Issues - Vendor App

### ISSUE THREAD

**Issue by kllittle** [CARLI/select#93]
When entering pricing, the filters work fine when you are trying to select libraries.  When trying to price individual libraries the radio buttons are adding and removing libraries from the list.  You cannot select a specific set of libraries for pricing.

**Comment by kllittle** [comment]
There is still a problem with trying to select Per FTE pricing for individual libraries. College of Dupage is appearing and disappearing when you are trying to select the radio button.

**Comment by jennyem56** [comment]
If I masquerade as a new vendor (that was created in the stating server), I get a blank screen.  This makes me believe that the new vendors I've created (and got an error for, despite showing up in the staff vendor list), are not really there.

**Comment by nicolemoorepixo** [comment]
Hi Katrina, 

Can you provide screenshots and steps to reproduce the error you're seeing with adding and removing libraries? Thanks!

**Comment by CClennon** [comment]
I just tried to masquerade as a vendor to enter pricing and am also getting the blank screen Jenny reported 18 days ago. I also tried refreshing my screen and this did not help.  Here is what I see

![image](https://user-images.githubusercontent.com/33095526/52237773-0ffab100-2890-11e9-9a28-8b936e103d5c.png)

**Comment by nicolemoorepixo** [comment]
Hey Cindy! Can you send me the URL from your screenshot and what vendor you tried to masquerade as?

**Comment by CClennon** [comment]
Hi Nicole,

The URL is http://select-vendor-stage.carli.illinois.edu/dashboard and the vendor is Joanne Fabric.  They are a new vendor.  Katrina was just in my office and had me close and reopen their pricing and refresh again when trying to masquerade – still nothing.

Thanks,

Cindy

Cindy Clennon
Director of Electronic Resources
Consortium of Academic and Research Libraries in Illinois (CARLI)
100 Trade Centre Drive, Suite 303
Champaign, IL 61820-7233
Ph: 217-333-4895
Fax: 217-244-7596
clennon@uillinois.edu<mailto:clennon@uillinois.edu>
www.carli.illinois.edu<http://www.carli.illinois.edu/>

"Under the Illinois Freedom of Information Act (FOIA), any written communication to or from university employees regarding University business is a public record and may be subject to public disclosure."

**Comment by CClennon** [comment]
I got an error message when I created the new vendor that was similar to the one you saw, but then when I refreshed it was included in the vendor list.  I was able to create a product and license agreement attached to that vendor, but was not able to proceed with masquerading.

**Comment by CClennon** [comment]
I just went through the process of creating a new vendor and I grabbed the error messages I saw:

This is the error message I saw when initially creating a new vendor, InfoBase:
{"error":"Could not create database cycle-sfx-maintenance-2019-04ab8ed6-60a5-41ef-98f4-c2f5fc2d925d statusCode=412","message":null,"originalError":{"error":"Could not create database cycle-sfx-maintenance-2019-04ab8ed6-60a5-41ef-98f4-c2f5fc2d925d statusCode=412"},"statusCode":500}

Then when I went in to masquerade as Infobase I saw the following message:
{"error":"no_db_file","message":null,"originalError":{"error":"no_db_file"},"statusCode":404}

I am still seeing the blank screen when I try to masquerade.

**Comment by nicolemoorepixo** [comment]
Update on this ticket:
Some of the vendor issues noted above can be retested now that the cycle creation issue has been fixed.

The radio buttons removing libraries was an issue Katrina reproduced for us during our call and remains open. I'll make an individual ticket for that one in JIRA

**Comment by CClennon** [comment]
Retesting the issue of the quick pricing tool changing the selected libraries and the problems continue.  Here are the steps I followed (no screen captures as they include pricing):
1) I selected ProQuest's eBrary Academic Complete
2) the following libraries were selected: Harper College, Heartland Community College and Highland Community College.  The initial pricing option shows "Percentage price Increase".  
3) I tried to change the price option to "Dollar amount". The radio button did not change and now Chicago State University has been added to the libraries list.
4) I tried to change the price option to "Price per FTE".  Again the radio button did not change but College of DuPage was added to the libraries list.
5) I tried to change the price option to "No pricing offered".  Again the radio button did not change and College of Lake County is now included in the list.

**Comment by CClennon** [comment]
I can send the Word document with the screen captures in it separately if that would be helpful.

**Comment by nicolemoorepixo** [comment]
I don't think we need the screen captures yet since we saw it reproduced in our call and have that recording. I'll let you know if that doesn't get us enough info and then we can get some screen captures from you.

**Comment by nicolemoorepixo** [comment]
Transferred to JIRA as CS-11
